docs/troubleshooting.asciidoc

```diff
@@ -115,9 +115,7 @@ Make sure that the agent is *both required and started at the very top of your m
 It's important that the agent is started before any other modules are
 `require`d.  If not, the agent will not be able to hook into any modules and
 will not be able to measure the performance of your application.
-
-IMPORTANT: If you are using Babel / ES modules in your application,
-make sure you have read about the <<es-modules,Babel / ES Module support>>.
+See <<start-gotchas>> for a number of possible surprises in correctly starting the APM agent when used with some compilers or bundlers.
```

Comment:
Are there other reasons users would end up on the troubleshooting page and need to see the start gotchas? Is this link prominent enough?

Reply:
tl;dr: The only side-effect I can think of right now of starting the agent too early (i.e. hitting a start gotcha) is that the user doesn't get expected auto-instrumentation. So, mostly, I think "no". This is the main reason to end up here.

However, fair question. I think this section of the troubleshooting page could be improved a lot.  Truth be told I ran out of steam on this PR (I was many more days into than I had planned for).  I was going for the minimum change to replace the `<<es-modules>>` link that was dropped.

Longer term I'd like to attempt https://github.com/elastic/apm-agent-nodejs/issues/2578 to have a relatively prominent log warning if we can detect a too-late agent start.  As well (and perhaps after that), this troubleshooting doc could quote examples of those log warnings so theoretically a google search for particular warnings would hit this page that could have a better explanation.

Reply:
Fair enough. I think this is fine for now.